### PR TITLE
Fix carriage return inside commented python code

### PR DESCRIPTION
**Pull request by vincent-legoll**
This looks more conventionnal

### CODE REVIEW

File: doc/source/reference/arrays.ndarray.rst [hunk 1]
```diff
@@ -46,7 +46,7 @@ objects implementing the :class:`buffer` or :ref:`array
    The array can be indexed using Python container-like syntax:
 
    >>> x[1,2] # i.e., the element of x in the *second* row, *third*
-   column, namely, 6.
```
Comment (charris) [inline doc/source/reference/arrays.ndarray.rst]: How about
```
>>> # The element of x in the *second* row, *third* column, namely, 6.
```
preceding the example?

**Comment by charris** [comment]
See commit message formatting at http://docs.scipy.org/doc/numpy-1.10.1/dev/gitwash/development_workflow.html. You can rewrite commit messages with `git rebase -i` followed by a force push.

File: doc/source/reference/arrays.ndarray.rst (hunk 1 above)
Comment (vincent-legoll) [inline doc/source/reference/arrays.ndarray.rst]: That would be OK for me, go ahead

Comment (charris) [inline doc/source/reference/arrays.ndarray.rst]: No, no, that is for you ;) I just do the reading and merges.

**Pull request closed without merging**